docs/api.md

```diff
@@ -2142,6 +2142,7 @@ await page.waitForRequest(request => request.url().searchParams.get('foo') === '
 ```js
 const firstResponse = await page.waitForResponse('https://example.com/resource');
 const finalResponse = await page.waitForResponse(response => response.url() === 'https://example.com' && response.status() === 200);
+const finalResponse = await page.waitForResponse(async (response) => { await response.text().includes('<html>') });
```

Comment:
Can we come up with a more realistic example?
Also, you probably want to name the variable something else but `finalResponse` and most likely meant to do `(await response.text()).includes(...)` instead of calling `includes` on the `Promise` object?